webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/repository/NodeGroupRepository.scala

```diff
@@ -190,6 +190,7 @@ final case class FullNodeGroupCategory(
       case FullOtherTarget(t) => t match {
         case AllTarget => true
         case AllTargetExceptPolicyServers => !node.isPolicyServer
+        case AllPolicyServers => node.isPolicyServer
```

Comment:
this file should not change

Reply:
it removes a warning missing pattern match (and the corresponding runtime exception) for a previous pr regarding migration. 
I'm adding a comment in head.

Reply:
ok